src/Yarhl/IO/BinaryFormat.cs

```diff
@@ -93,6 +94,15 @@ public BinaryFormat(Stream stream, long offset, long length)
             private set;
         }
 
+        /// <inheritdoc />
```

Comment:
Can you add in the remarks of the documentation of this method that it creates the copy in memory (good to know for large streams).

Reply:
Done!